src/dbcp/transform/ballot_ready.py

```diff
@@ -155,11 +184,12 @@ def _explode_counties(raw_ballot_ready: pd.DataFrame) -> pd.DataFrame:
 
     # Valdez-Cordova Census Area was split into two areas in 2019
     # https://www.census.gov/programs-surveys/geography/technical-documentation/county-changes/2010.html
-    # All elections are state and federal level so I will duplicate the races for the two new census areas
+    # It is reasonable to split State and Federal elections between the two areas.
+    # However, adding local elections to both counties is not appropriate. I'm going to do it
+    # anyways because there aren't any great options for accurately geocoding the local elections.
     valdez = ballot_ready.query("county_id_fips == '02261'")
-    assert valdez.level.isin(
-        ["state", "federal"]
-    ).all(), "Found a local election in the Valdez-Cordova Census Area!"
+    if valdez.level.isin(["state", "federal"]).all():
+        logger.info("Found a local election in the Valdez-Cordova Census Area!")
```

Comment:
I figured it's better to have duplicates than to lose these elections or do a bad job at geocoding them.